src/cljs/time_tracker_web_nxt/views.cljs

```diff
@@ -17,35 +17,40 @@
 (defn element-value [event]
   (-> event .-target .-value))
 
-(defn project-dropdown [projects selected]
-  (let [selected-id (:id @selected)]
-    [:select.project-dropdown
-     {:value (or selected-id "")
-      :on-change #(reset! selected {:id (-> % .-target .-value)})}
-     (for [{:keys [id name]} projects]
-       ^{:key id} [:option (if (= selected-id id)
-                             {:value id :label name}
-                             {:value id :label name}) name])]))
+(defn dropdown-widget [items selected-id event]
+  [:select.project-dropdown
+   {:value (or selected-id "")
+    :on-change (fn [val]
+                 (rf/dispatch [event (-> val element-value int)]))}
+   (for [{:keys [id name]} items]
+     ^{:key id} [:option (if (= selected-id id)
+                           {:value id :label name}
+                           {:value id :label name}) name])])
 
 (defn create-timer-widget []
-  (let [projects         (rf/subscribe [:projects])
-        default          {:id (:id (first @projects))}
-        selected-project (reagent/atom default)
-        data            (reagent/atom {:notes ""
+  (let [data            (reagent/atom {:notes ""
                                        :elapsed-hh 0
                                        :elapsed-mm 0
-                                       :elapsed-ss 0})
-        show?            (rf/subscribe [:show-create-timer-widget?])]
+                                       :elapsed-ss 0})]
     (fn []
-      (let [default-selected     {:id (:id (first @projects))}
+      (let [show?            (rf/subscribe [:show-create-timer-widget?])
```

Comment:
In general, it's a good practice to align the bindings inside a let for the sake of readability.

Reply:
@kirang89 fixed all the alignment issues in https://github.com/nilenso/time-tracker-web-nxt/pull/44